drkafka/src/main/java/com/pinterest/doctorkafka/api/MaintenanceApi.java

```diff
@@ -0,0 +1,53 @@
+package com.pinterest.doctorkafka.api;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import com.pinterest.doctorkafka.DoctorKafka;
+import com.pinterest.doctorkafka.KafkaClusterManager;
+
+@Path("/cluster/{clusterName}/admin/maintenance")
```

Comment:
update  README.md with these rest APIs and how we shall use them?

Reply:
Added a separate page docs/APIs.md to capture API docs.